encodings/fastlanes/src/bitpacking/compute/search_sorted.rs

```diff
@@ -20,7 +20,7 @@ impl SearchSortedFn for BitPackedArray {
             let unwrapped_value: $P = value.cast(self.dtype())?.try_into().unwrap();
                 if let Some(patches_array) = self.patches() {
                 if unwrapped_value as usize >= self.max_packed_value() {
-                    Ok(search_sorted(&patches_array, value.clone(), side)?.map(|i| i - self.offset()))
+                    search_sorted(&patches_array, value.clone(), side)
```

Comment:
This was a bug in SparseArray search_sorted that I worked around in bitpacking but also independently fixed it in SparseArray